### PR TITLE
Add rocket 2 doc updates

### LINKED ISSUE #61: document Rocket2
Our Slack guidance still talks about old Rocket - would be good if we could get this updated with a simple "getting started" for Rocket 2!

/cc @cheukyin699 maybe? :) https://docs.ubclaunchpad.com/handbook/tools/slack#rocket

### PR DESCRIPTION
**Pull request by cheukyin699**
# Related Tickets

*List relevant tickets, e.g. 'Closes #<some_ticket_number>'*

Closes #61

# Changes

*Briefly describe the changes you made*

Add slightly newer documentation on the new Rocket 2 system.

### CODE REVIEW

File: handbook/tools/slack.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ of each category of channel by searching for a prefix!
   for Launch Pad leadership
 * [`#ask-design`](https://ubclaunchpad.slack.com/messages/C747550BD): Questions
   for Launch Pad's design teams
+* `#ask-rocket`: If rocket isn't working, use it to express your frustrations
```
Comment (bobheadxi) [inline handbook/tools/slack.md:76]:
```suggestion
* [`#ask-rocket`](https://ubclaunchpad.slack.com/messages/CK93HTYQN): If rocket isn't working, use it to express your frustrations
```

File: handbook/tools/slack.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ of each category of channel by searching for a prefix!
   for Launch Pad leadership
 * [`#ask-design`](https://ubclaunchpad.slack.com/messages/C747550BD): Questions
```
Comment (bobheadxi) [inline handbook/tools/slack.md:74]: Unrelated, but do you mind tacking this to this PR?

```suggestion
* [`#ask-design`](https://ubclaunchpad.slack.com/messages/CK0H2GNQH): Questions
```

File: handbook/tools/slack.md [hunk 3]
```diff
@@ -82,25 +83,18 @@ of each category of channel by searching for a prefix!
 
 ### Rocket
```
Comment (bobheadxi) [inline handbook/tools/slack.md:84]:
```suggestion
### Rocket <Badge type="tip" text="updated"/>
```

File: handbook/tools/slack.md (hunk 2 above)
Comment (cheukyin699) [reply handbook/tools/slack.md:74]: sure

**Pull request merged**